common/src/IO/TextureReader.cpp

```diff
@@ -61,7 +61,15 @@ namespace TrenchBroom {
 
         std::string TextureReader::PathSuffixNameStrategy::doGetTextureName(const std::string& /* textureName */, const Path& path) const {
             if (m_prefixLength < path.length()) {
-                return path.suffix(path.length() - m_prefixLength).deleteExtension().asString("/");
+				if (path.asString().find(".D") != std::string::npos) { //we have a mip texture, probably, so remove extension
```

Comment:
I don't think we should change this at all. The changes should be limited to the obj exporter only. TrenchBroom uses forward slashes internally everywhere, if you change this here, it may affect other parts of the code that assume that pathes have forward slashes.